packages/bot-web-ui/src/stores/app-store.ts

```diff
@@ -49,17 +49,35 @@ export default class AppStore {
         link: localize('Switch to another account'),
     });
 
-    getErrorForEuClients = (is_logged_in = false) => {
+    getErrorForEuClients = (is_logged_in = false, country: string | undefined = undefined) => {
         return {
             text: ' ',
             title: is_logged_in
-                ? localize('Deriv Bot is not available for EU clients')
-                : localize('Deriv Bot is unavailable in the EU'),
+                ? localize(`Deriv Bot is not available for ${country || 'EU'} clients`)
+                : localize(`Deriv Bot is unavailable in ${country || 'the EU'}`),
             link: is_logged_in ? localize("Back to Trader's Hub") : '',
             route: routes.traders_hub,
         };
     };
 
+    throwErrorForExceptionCountries = (client_country: string) => {
+        const { client, common } = this.core;
+        const not_allowed_clients_country = [
```

Comment:
why did we do with data structure and not a simple object with key value pair? and also can we add the country list to the same file where we have added EU countries

Reply:
@maryia-matskevich-deriv just checked , eu list is in core ,now,  I will not recommend adding it there. For now then list can stay here or if you find a better place to add it please move.

Reply:
i've changed not_allowed_clients_country from array to object✅